index.html

```diff
@@ -1,22 +1,237 @@
+							alt="This should be an user icon">
+					</div>
+					<div class="comment-info-container">
+						<div class="comment-first-part">
+							<p class="user-name">Santiago Velosa</p>
+							<p class="comment-time">A day ago</p>
+						</div>
+						<p class="actual-comment">This is a very inspiring story, thank you so much for sharing, Rick!
+						</p>
+						<div class="comment-interaction">
+							<div class="likes">
+								<i class="far fa-thumbs-up"></i>
+								<p>648</p>
+							</div>
+							<div class="comment-response">
+								<i class="far fa-thumbs-down"></i>
+								<p>REPLY</p>
+							</div>
+						</div>
+					</div>
+				</div>
+			</section>
+		</section>
+		<section id="thumbnails">
+			<div id="thumbnail-container">
+				<ul>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=O_LVl8w4-vw" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/O_LVl8w4-vw/mqdefault_6s.webp?du=3000&sqp=CPTpq4sG&rs=AOn4CLDGr1zzvlqIiI6H7YJ-o3tMYUtmSA"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">The Future of This Channel</p>
+								<p class="thumbnail-description">
+									Rick Beato
+									<br>
+									511,370 views
+									<br>
+									A month ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=NaqySto4bKI" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/NaqySto4bKI/mqdefault_6s.webp?du=3000&sqp=CMqXrIsG&rs=AOn4CLArcSYGrOHfxlGwL8mn03xl_FKf_A"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">How I Sold 1,000,000 Records</p>
+								<p class="thumbnail-description">
+									Rick Beato
+									<br>
+									369,725 views
+									<br>
+									two months ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=k2vqJ78VA4g" target="_blank">
+								<img id="img"
+									src="https://i.ytimg.com/an_webp/k2vqJ78VA4g/mqdefault_6s.webp?du=3000&sqp=COiOrIsG&rs=AOn4CLBSyTdE6kqQ_fssYjfPPcvNj4oy6g"
+									alt="This should be a thumbnail image"></a>
+							<div class="description-container">
+								<p class="thumbnail-title">The Ron Carter Interview</p>
+								<p class="thumbnail-description">
+									Rick Beato
+									<br>
+									231,778 views
+									<br>
+									two months ago
+								</p>
+							</div>
+						</div>
+					</li>
+					<li class="mini-container">
+						<div class="thumbnail-video-container">
+							<a href="https://www.youtube.com/watch?v=mgkvU5eCaYo" target="_blank">
+								<img id="img"
```

Comment:
You cannot have more than one element with the same id in an HTML document. Please change it to class